index.html

```diff
@@ -5,12 +5,33 @@
         <title>ubio application bundle</title>
         <link href="/src/css/index.css" rel="stylesheet">
         <script type="module">
-            window.location.hash = '';
-            localStorage.clear();
-
             import { createEndUserSdk } from '/web_modules/@ubio/sdk.js';
             import { createApp } from '/src/main.js';
 
+            async function getExisting() {
+                const existingJob = localStorage.getItem(`job-session`);
```

Comment:
later in code it's `job-session-${domain}` https://github.com/universalbasket/sdk-application-bundle/pull/65/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR48 
intentional?

Reply:
I need to remove this. The problem is that storage elsewhere doesn't distinguish.